### PR TITLE
Fix a few return types in stdlib/2/inspect.pyi.

### PR DESCRIPTION
**Pull request by rchen152**

### CODE REVIEW

File: stdlib/2/inspect.pyi [hunk 1]
```diff
@@ -81,9 +87,9 @@ Traceback = NamedTuple(
 
 _FrameRecord = Tuple[FrameType, str, int, str, List[str], int]
```
Comment (gvanrossum) [inline stdlib/2/inspect.pyi]: I'd also rename this to `_FrameInfo` since in Python 3 it's a named tuple definition named `FrameInfo`.

**Comment by gvanrossum** [comment]
Thanks! Just one suggestion.

**Comment by rchen152** [comment]
Changed!

**Comment by gvanrossum** [comment]
Thanks!!

**Pull request merged**